packages/react/src/toggle-group/ToggleGroup.tsx

```diff
@@ -0,0 +1,20 @@
+import * as RadixToggleGroup from "@radix-ui/react-toggle-group";
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+
+type ToggleGroupProps = Omit<BoxProps<typeof RadixToggleGroup.Root>, "type">;
```

Comment:
why are we omitting `type` here?

Reply:
it supports multiple where we can toggle multiple buttons of the group. 
To me, it seems like we're going for toggling only one button among the group. For this reason, we're omitting `type`

Reply:
let's keep the `type` here - so consumers can decide whether they want single or multiple

we can default it to single from our side - so something like:
```tsx
export function ToggleGroup({ type = "single" }) {
  return <RadixToggleGroup.Root type={type} />
}
```